RadixWallet/Features/ClaimWallet/ClaimWallet+Reducer.swift

```diff
@@ -34,13 +30,13 @@ public struct ClaimWallet: Sendable, FeatureReducer {
 	public func reduce(into state: inout State, viewAction: ViewAction) -> Effect<Action> {
 		switch viewAction {
 		case .clearWalletButtonTapped:
-			.run { send in
+			return .run { send in
 				await resetWalletClient.resetWallet()
 				await send(.delegate(.didClearWallet))
 			}
 		case .transferBackButtonTapped:
-			// TODO: transfer back
-			.send(.delegate(.didTransferBack))
+			state.isLoading = true
```

Comment:
Is a loading state necessary on this screen, considering that the actual profile claiming is handled by `CloudBackupClient` and tapping this button will immediately dismiss the screen?

Reply:
No it's not needed anymore. I used to have more loading state handling but apparently i forgot to remove the last bits

Reply:
I actually kept it, but on the other button. In theory resetWallet could take time and we don't want them to be able to keep tapping